stress_test/controller_test_basic.py

```diff
@@ -11,100 +11,142 @@
+datastore_conf_path = os.path.join(ctrl_base_dir, 'distribution-karaf-0.4.0-'
+                                   'Beryllium/etc')
 
 if ctrl.persistence_hnd:
-    #disable persistence
+    '''disable persistence'''
     ctrl.disable_persistence()
 
-    #check the effect of disable_ persistence()
-    path_file = os.path.join(datastore_conf_path,'org.opendaylight.controller.cluster.datastore.cfg')
+    '''check the effect of disable_ persistence'''
+    path_file = os.path.join(datastore_conf_path, 'org.opendaylight.'
+                             'controller.cluster.datastore.cfg')
     pattern = 'persistent=false'
 
     cmd = ('grep {0} {1}'.format(pattern, path_file))
-    exit_status, output = util.netutil.ssh_run_command(ctrl._ssh_conn, cmd,'Check_persistence')
-
-    if (output!= None):
-        logging.info ("[Testing] Persistence is disabled successfully")
+    exit_status, output = util.netutil.ssh_run_command(ctrl._ssh_conn,
+                                                       cmd,
+                                                       'Check_persistence')
+    if (output is not None):
+        logging.info('[Testing] Persistence is '
+                     'disabled successfully')
     else:
-        logging.info ("[Testing] Persistence is still enabled")
+        logging.info('[Testing] Persistence is still enabled')
 
 ctrl.generate_xmls()
-try:
 
-    #start a controller
+try:
+    '''start a controller'''
     ctrl.check_status()
     ctrl.start()
-
-    #change stat period
+    '''change stat period'''
     ctrl.change_stats()
-
-    #check the effect of change_stats()
-    path_file = os.path.join(datastore_conf_path,'opendaylight','karaf','30-statistics-manager.xml')
-    pattern = '<min-request-net-monitor-interval>'+str(ctrl.stat_period_ms[0])+'</min-request-net-monitor-interval>'
+    '''check the effect of change_stats()'''
+    path_file = os.path.join(datastore_conf_path,
+                             'opendaylight',
+                             'karaf',
+                             '30-statistics-manager.xml')
+    pattern = '<min-request-net-monitor-'
+    'interval>' + str(ctrl.stat_period_ms[0]) + '</min-request-net-'
+    'monitor-interval>'
 
     cmd = ('grep {0} {1}'.format(pattern, path_file))
-    exit_status, output = util.netutil.ssh_run_command(ctrl._ssh_conn, cmd,'Check_statistics_period')
-    if (output!= None):
-        logging.info ("[Testing] Interval statistics has been updated successfully")
+    exit_status, output = util.netutil.ssh_run_command(ctrl._ssh_conn,
+                                                       cmd,
+                                                       'Check_stats_period')
+    if (output is not None):
+        logging.info('[Testing] Interval statistics have been '
+                     'updated successfully')
     else:
-        logging.info ("[Testing] Interval statistics not updated")
-
+        logging.info("[Testing] Interval statistics not updated")
+
```

Comment:
yes, it is has been tested by the means of the controller_test_basic.py on a Mininet topology with 5 switches, 5 hosts, 10 links and 13 flows